.github/workflows/signjars.yml

```
@@ -0,0 +1,92 @@
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    # Step 1: Sign JARs and set SIGNED_JARS_DIR as output
+    - name: Codesign JARs and Internal Native Libraries
+      id: sign_jars  # Assign an id to reference outputs
+      env: 
+        MACOS_CERTIFICATE: ${{ secrets.MACOS_CERTIFICATE }}
+        MACOS_CERTIFICATE_PWD: ${{ secrets.MACOS_CERTIFICATE_PWD }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2162:info:29:64: read without -r will mangle backslashes

(shellcheck)